libtiledbvcf/cmake/Modules/FindSpdlog.cmake

```diff
@@ -73,8 +74,8 @@ if (NOT SPDLOG_FOUND)
       PREFIX "externals"
       # Set download name to avoid collisions with only the version number in the filename
       DOWNLOAD_NAME ep_spdlog.zip
-      URL "https://github.com/gabime/spdlog/archive/v1.9.0.zip"
-      URL_HASH SHA1=6259d1b6c5b9b565aa3ba5a6315d49f76d90ec0a
+      URL "https://github.com/gabime/spdlog/archive/v1.11.0.zip"
```

Comment:
Does this update need to be down in this PR?

Reply:
IIRC, can live with old version -if- are not building with DOWNLOAD_TILEDB_PREBUILT=OFF, 
i.e. if -not- building tiledb from source, just fetching binary (the default is '=ON'), but build fails if building tiledb from source.